Riot/Modules/Application/LegacyAppDelegate.m

```diff
@@ -3726,81 +3726,99 @@ - (void)checkPendingRoomKeyRequestsInSession:(MXSession*)mxSession
         return;
     }
 
+    MXWeakify(self);
     [mxSession.crypto pendingKeyRequests:^(MXUsersDevicesMap<NSArray<MXIncomingRoomKeyRequest *> *> *pendingKeyRequests) {
-
+        
+        MXStrongifyAndReturnIfNil(self);
         NSLog(@"[AppDelegate] checkPendingRoomKeyRequestsInSession: pendingKeyRequests.count: %@. Already displayed: %@",
-              @(pendingKeyRequests.count),
-              roomKeyRequestViewController ? @"YES" : @"NO");
+               @(pendingKeyRequests.count),
+              self->roomKeyRequestViewController ? @"YES" : @"NO");
```

Comment:
It would be useful to log `mxSession.crypto.crossSigning.state` to provide support in the future.

Reply:
State added to the log